packages/flutter/lib/src/semantics/semantics_event.dart

```diff
@@ -14,7 +14,10 @@ abstract class SemanticsEvent {
   /// Initializes internal fields.
   ///
   /// [type] is a string that identifies this class of [SemanticsEvent]s.
-  SemanticsEvent(this.type);
+
+  /// Abstract const constructor. This constructor enables subclasses to provide
+  /// const constructors so that they can be used in const expressions.
+  const SemanticsEvent(this.type);
```

Comment:
looks like this constructor actually has useful material so doesn't need the boilerplate (maybe a merge conflict?)